import_email_events.py

```diff
@@ -0,0 +1,22 @@
+#
+# Script to import "email event" metrics from S3 into redshift.
+#
+
+import import_events
+SCHEMA = """
+    type VARCHAR(30) NOT NULL ENCODE lzo,
+    locale VARCHAR(40) ENCODE lzo,
+    domain VARCHAR(40) ENCODE lzo,
+    template VARCHAR(64) ENCODE lzo,
+    flow_id VARCHAR(64) DISTKEY ENCODE lzo
+"""
+
+COLUMNS = "type, locale, domain, template, flow_id"
+
+import_events.run(s3_prefix="fxa-email",
```

Comment:
Correction, it should be `"fxa-email/data/email-events"`, I only just had a look at S3. However that also isn't working properly yet, we might need to generalise `import_events.py` a bit more as part of this PR. Digging into it now...